### ISSUE TITLE
Choice of validation

### ISSUE THREAD

**Issue by SSSilvester** [zhu-xlab/PFST#2]
Sorry，why are images from the training set used in the validation part of this document instead of images from the validation set?

    val=dict(
        type=dataset_type,
        data_root=data_root_pots,

>         img_dir='img_dir/train',
>         ann_dir='ann_dir/train',

        gt_seg_map_loader_cfg=gt_seg_map_loader_cfg,
        pipeline=test_pipeline),
    test=dict(
        type=dataset_type,
        data_root=data_root_pots,
        img_dir='img_dir/val',
        ann_dir='ann_dir/val',
        gt_seg_map_loader_cfg=gt_seg_map_loader_cfg,
        pipeline=test_pipeline),
